include/sdk/interfaces/schemasystem/schema.h

```diff
@@ -665,6 +725,10 @@ class CSchemaPtrMap {
 public:
     CUtlMap<K, V> m_Map;
     CThreadFastMutex m_Mutex;
+
+#if not defined(CS2) && not defined(DOTA2)
```

Comment:
please... we aren't coding in python
```suggestion
#if !defined(CS2) && !defined(DOTA2)
```

Reply:
https://www.youtube.com/watch?v=k13LlCabKdQ